tests/common/gcp_type/iam_policy_test.py

```diff
@@ -52,7 +52,8 @@ def setUp(self):
             'serviceAccount:abc@def.gserviceaccount.com',
             'user:someone@somewhere.tld',
             'allUsers',
-            'user:anything'
+            'user:anything',
+            'allAuthenticatedUsers'
```

Comment:
nit: to be consistent with `self.members`, please move this up one line, so that it's closer to `allUsers`?  You will need to update your test reference.

Reply:
Done